nms/app/views/organizations/__tests__/OrganizationEditTest.js

```diff
@@ -0,0 +1,302 @@
+        <MuiStylesThemeProvider theme={defaultTheme}>
+          <SnackbarProvider>
+            <AppContextProvider>
+              <Routes>
+                <Route path="organizations/*" element={<Organizations />} />
+                <Route
+                  path="organizations/detail/:name"
+                  element={<OrganizationEdit />}
+                />
+              </Routes>
+            </AppContextProvider>
+          </SnackbarProvider>
+        </MuiStylesThemeProvider>
+      </MuiThemeProvider>
+    </MemoryRouter>
+  );
+};
+
+const WrappedOrganizationsEdit = () => {
+  return (
+    <MemoryRouter
+      initialEntries={['/organizations/detail/magma-test']}
+      initialIndex={0}>
+      <MuiThemeProvider theme={defaultTheme}>
+        <MuiStylesThemeProvider theme={defaultTheme}>
+          <SnackbarProvider>
+            <AppContextProvider>
+              <Routes>
+                <Route
+                  path="organizations/detail/:name"
+                  element={<OrganizationEdit />}
+                />
+              </Routes>
+            </AppContextProvider>
+          </SnackbarProvider>
+        </MuiStylesThemeProvider>
+      </MuiThemeProvider>
+    </MemoryRouter>
+  );
+};
+
+describe('<OrganizationEdit />', () => {
+  it('renders with 2 organizations', async () => {
+    await axios.get.mockResolvedValueOnce({
+      data: {organizations: organizationsMock},
+    });
+    await axios.get.mockResolvedValueOnce({
+      data: usersMock,
+    });
+    await axios.get.mockResolvedValueOnce({
+      data: hostUserMock,
+    });
+    await axios.get.mockResolvedValueOnce({
+      data: ['test', 'test1', 'test2', 'test3', 'test4'],
+    });
+    await axios.get.mockResolvedValueOnce({
+      data: ['test', 'test1', 'test2', 'test3', 'test4'],
+    });
+    await axios.get.mockResolvedValueOnce({
+      data: {organization: {...organizationsMock[0]}},
+    });
+
+    const {getByTestId, getByText, queryByTestId, getAllByTitle} = render(
+      <WrappedOrganizations />,
+    );
+
+    await waitFor(() => {
+      expect(getByTestId('organizationTitle')).toBeInTheDocument();
+    });
+
+    //Onboarding Modal
+    expect(getByTestId('onboardingDialog')).not.toBeNull();
+    fireEvent.click(getByText('Get Started'));
+    expect(queryByTestId('onboardingDialog')).toBeNull();
+
+    // Open menu to go to organization detail page
+    const actionList = getAllByTitle('Actions');
+    expect(getByTestId('actions-menu')).not.toBeVisible();
+    fireEvent.click(actionList[0]);
+    expect(getByTestId('actions-menu')).toBeVisible();
+    await waitFor(() => {
+      fireEvent.click(getByText('View'));
+    });
+    expect(getByTestId('actions-menu')).not.toBeVisible();
+
+    // organizationEdit
+    expect(getByTestId('organizationEditTitle')).toHaveTextContent('host');
+  });
+
+  it('renders with 2 organizations', async () => {
+    // at fetchNetworks (app/views/organizations/OrganizationDialog.js:135:19)
```

Comment:
If those comments are supposed to stay then having the line number isn't very future proof